src/components/Button/Button.story.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select } from '@storybook/addon-knobs';
+import Button from './Button';
+
+const stories = storiesOf('Input', module);
```

Comment:
rename story to `Buttons`, Input should be reserved for other form elements like text inputs.